src/styles/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: $white;
+  padding: 16px;
+
+  &__img {
+    display: flex;
+    align-items: center;
+    width: 160px;
+    height: 134px;
+
+    &-wrapper {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      margin-bottom: 40px;
+      margin-top: 16px;
+    }
+  }
+
+  &__title {
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    display: block;
+    color: $secondary;
+    font-size: 10px;
+    font-weight: 400;
```

Comment:
You can set `font-weight: 400;` as a default font-weight to body or html element